### DISCUSSION TITLE
Remove keybinding for s key

### DISCUSSION THREAD

**Discussion by schrute99** [LunarVim/LunarVim#3838] [Q&A] [answered]
hey everyone,

I just set up lunarvim and noticed that when I press "s", the current character is deleted and the editor also switches into insert mode (so basically like a combination of "x" and "i").
I tried to remove this binding with `lvim.lsp.buffer_mappings.normal_mode['s'] = nil` but it's still there.
Has someone managed to remove this binding?

Edit:
my version is
```
NVIM v0.9.0-dev-920+gc9b0fe1f4
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
```
and I'm using Linux Mint.

Thanks!

**Comment by LostInTheLogs** [comment, marked as answer]
it's a default vim binding, see `:help s`
